.gitignore

```diff
@@ -2,3 +2,4 @@
 .Rhistory
 .RData
 .Ruserdata
+\playground
```

Comment:
Don't forget to remove the playground folder again (and also remove it from the .gitignore then).